lib/command.js

```diff
@@ -1308,6 +1309,36 @@ Expecting one of '${allowedValues.join("', '")}'`);
     }
   }
 
+  /**
+   * Display an error message if exclusive options are used together.
+   *
+   * @api private
+   */
+  _checkForExclusiveOptions() {
+    const definedNonDefaultOptions = this.options.filter(
+      (option) => {
+        const optionKey = option.attributeName();
+        if (this.getOptionValue(optionKey) === undefined) {
+          return false;
+        }
+        return this.getOptionValueSource(optionKey) !== 'default';
+      }
+    );
+
+    const optionsWithExclusiveNames = definedNonDefaultOptions.filter(
+      (option) => option.exclusiveNames.length > 0
+    );
+
+    optionsWithExclusiveNames.forEach((option) => {
+      const exclusiveAndDefined = definedNonDefaultOptions.filter((defined) =>
```

Comment:
Observation rather than suggestion: this could be done using `find`, since we only care about one. (I don't mind the filter in part because of the previous filters, in that mindset!)

Reply:
Right! At first I thought about reporting all conflicting options and saw it was too confusing (other tools also report the first one).

Fixed in https://github.com/tj/commander.js/pull/1678/commits/744a3396086477867d889a48eb13c41837e8a2f3